src/App.tsx

```diff
@@ -100,17 +104,43 @@ const Tabs = ({ navigation, route }) => {
 };
 
 const App = () => {
+  const [inderLoaded] = useFonts({
```

Comment:
Is there a way we can abstract this fonts checking to another file? Or encapsulate it in an object/helper function?

Reply:
yea, definitely. It's a lame excuse but I had to hurry and get this done as I'm pretty busy currently :D